javascript/javascript1/week1/java.js

```diff
@@ -0,0 +1,103 @@
+// Age-ify (A future age calculator)
+const yearOfMyBirth = 1986;
+const yearFuture = 2035;
+const age = yearFuture - yearOfMyBirth;
+console.log("My Firend will be " + age + " years old in " + yearFuture);
+// Goodboy-Oldboy (A dog age calculator)
+const dogyearOfbirth = 2020;
+const dogyearFuture = 2028;
+let dogage = dogyearFuture - dogyearOfbirth;
+let typeofyaer = "human";
+console.log ("My Firend's Dog will "   +   dogage + " years ");
+
+//  result in dog years or human years
+const shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears) {dogage = 20
+typeofyaer = "dog"}
+console.log (shouldShowResultInDogYears);
+
+
+// Housey pricey (A house price estimator)
+// Peter home
+const Peter = [
+    
+        houseWide = 8,
+        houseDeep = 10,
+        houseHigh = 10,
+        gardenSizeInM2 =  100,
+        housecost = 2500000,
+]
+let volumeInMetersPeter ;
+volumeInMeterspeter = houseWide * houseDeep * houseHigh;
```

Comment:
No need to initiated a variable in one line, and then set it in the next one. You have also typed `peter` with small letter in one place and capital in the second. This could have been avoided if you used to autocomplete suggestions from VS Code.
```suggestion
let volumeInMetersPeter = houseWide * houseDeep * houseHigh;
```